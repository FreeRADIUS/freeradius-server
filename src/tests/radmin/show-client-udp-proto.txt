show client config 127.0.2.1
