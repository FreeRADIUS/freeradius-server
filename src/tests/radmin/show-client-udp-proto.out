shortname	paodequeijo_test_udp_client
secret		testing123
proto		udp
